gallery/how_to/work_with_microtvm/micro_pytorch.py

```diff
@@ -93,11 +93,11 @@
 # would run on a physical microcontroller.
 #
 
-
 # Simulate a microcontroller on the host machine. Uses the main() from `src/runtime/crt/host/main.cc`
-# To use physical hardware, replace "host" with another physical micro target, e.g. `nrf52840`
-# or `mps2_an521`. See more more target examples in micro_train.py and micro_tflite.py tutorials.
-target = tvm.target.target.micro("host")
+# To use physical hardware, replace `board` with another physical micro target, e.g. `nrf5340dk_nrf5340_cpuapp`
+# or `mps2_an521` and change the platform type to Zephyr.
+# See more more target examples in micro_train.py and micro_tflite.py tutorials.
```

Comment:
Can we mention these tutorials in text instead of in a code comment? That way we can add clickable links.

Reply:
good catch, fixed it